src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/TaskList/TaskListComponents.tsx

```diff
@@ -113,7 +113,7 @@ export const AssigneeEdit = (params: GridRenderEditCellParams) => {
       id: number;
     }[]
   ) => {
-    const teamMembers = team.members.concat(team.leader);
+    const teamMembers = team.members.concat(team.head);
```

Comment:
make a function that gets all the users on the team and then use it here and above.